testsuite/cluster-test/src/tx_emitter.rs

```diff
@@ -408,10 +408,14 @@ impl TxEmitter {
         let balance = retrieve_account_balance(&client, faucet_account.address).await?;
         for b in balance {
             if b.currency.eq(XUS_NAME) {
-                info!(
-                    "DD account current balances are {}, requested {} coins",
-                    b.amount, coins_total
-                );
+                if b.amount < coins_total {
+                    warn!("DD account current balance {} is low, requested {} coins", b.amount, coins_total);
```

Comment:
or error!.  Will leave this to you.

Reply:
we do not need error to crash the system, because if this is not the first time mint to child vasps, those vasps are reusable and I assume they already have some coins and balance is not 0. So for this case we only need to care about the first time mint to new child accounts